src/templates/cmpapi.js

```diff
@@ -98,32 +103,36 @@ window.__cmpapi = function (command, version, callback, parameter) {
       image.onerror = function () {
         log(logEvents.SET_CONSENT, false, {});
       };
+
       !!callback && callback(parameter);
       break;
     case 'removeConsentDecision':
-      image = document.createElement('img');
       localStorageAvailable = false;
+
+      image = document.createElement('img');
       image.src = '<%-CONSENT_SERVER_PROTOCOL%>://<%-CONSENT_SERVER_HOST%>/<%-API_VERSION%>/remove-consent';
       if (window.localStorage && localStorage.removeItem) {
         localStorage.removeItem('<%-COOKIE_NAME%>');
         localStorageAvailable = true;
       }
+
       image.onload = function () {
         log(logEvents.REMOVE_CONSENT_DECISION, true, { localStorageAvailable: localStorageAvailable });
       };
       image.onerror = function () {
         log(logEvents.REMOVE_CONSENT_DECISION, false, {});
       };
+
       !!callback && callback();
       break;
     case 'onLogEvent':
-      logCallback = callback;
+      logCallbacks[logCallbacks.length] = callback;
```

Comment:
So this is the main change, right? Having this as a list instead of a single callback.

Reply:
correct